diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+the backend is the database that users access for applications
+the backend stores the information of the application
+the backend is the framework, it's responsible for making everything on the frontend work
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+the model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+views is the presentation of the data
+views is only what's displayed on the screen
```

Comment:
`Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.`